wkcuber/export_wkw_as_tiff.py

```diff
@@ -58,56 +78,95 @@ def wkw_name_and_bbox_to_tiff_name(name: str, slice_index: int) -> str:
     if name is None or name == "":
         return f"{slice_index}.tiff"
     else:
-        return f"name_{slice_index}.tiff"
+        return f"{name}_{slice_index}.tiff"
+
+
+def wkw_slice_to_image(data_slice: np.ndarray):
+    # discard the z dimension
+    data_slice = data_slice.squeeze(axis=3)
+    if data_slice.shape[0] == 1:
+        # discard greyscale dimension
+        data_slice = data_slice.squeeze(axis=0)
```

Comment:
What would happen if you didn't do this? Would it create a multi-channel tiff file even though there is only one channel? I would have thought that this isn't a problem :thinking:

Reply:
The tests do fail if I remove the check and I can remember that the method only accepts  `m x n` or `m x n x 3` images but I cannot find the spot in the doc anymore.